bindings/csharp/Benchmarks/Benchmarks.csproj

```diff
@@ -16,7 +16,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Regorus" Version="0.6.0$(RegorusPackageVersionSuffix)"/>
+    <PackageReference Include="Regorus" Version="0.8.0$(RegorusPackageVersionSuffix)"/>
```

Comment:
>8.0 [](http://example.com/codeflow?start=51&length=3)

should we bump the version of the C# bindings to 0.8.1 so we can publish a new nuget with your latest changes?

Reply:
Sounds good :)